core/thread.c

```diff
@@ -171,7 +171,9 @@ void thread_add_to_list(list_node_t *list, thread_t *thread)
 #ifdef DEVELHELP
 uintptr_t thread_measure_stack_free(char *stack)
```

Comment:
```suggestion
uintptr_t thread_measure_stack_free(const char *stack)
```

Want to also make that `const` while you are at it?

Reply:
Done